src/documentation/index.tsx

```diff
@@ -113,11 +113,12 @@ const routes = [
   name: "Documentation",
 })
 class Documentation extends BaseComponent {
-  showSidebar = false;
+  showSidebar = true;
```

Comment:
What is showSidebar used for?

Reply:
I think nothing at all. To be honest i don't know why i changed it. 
I'll remove it completely.